packages/source-contentful/index.js

```diff
@@ -108,8 +110,8 @@ class ContentfulSource {
     }
   }
 
-  async fetch (method, limit = 1000, order = 'sys.createdAt') {
-    const fetch = skip => this.client[method]({ skip, limit, order })
+  async fetch (method, limit = 1000, order = 'sys.createdAt', parameters = {}) {
+    const fetch = skip => this.client[method]({ skip, limit, order, ...parameters })
```

Comment:
```suggestion
    const fetch = skip => this.client[method]({ ...parameters, skip, limit, order })
```
Just so `parameters` doesn't override the internal parameters.